install.sh

```diff
@@ -71,15 +98,24 @@ pacman --noconfirm -S \
 	python \
 	vulkan-icd-loader \
 	lib32-vulkan-icd-loader \
-	vulkan-radeon \
-	lib32-vulkan-radeon \
-	steam
+	steam \
+	steam-native-runtime
 
-# install nvidia graphics if needed
+# install NVIDIA graphics
 if lspci | grep -E -i '(vga|3d)' | grep -i nvidia > /dev/null; then
 	pacman --noconfirm -S nvidia nvidia-utils lib32-nvidia-utils
 fi
 
+# install AMD graphics
+if lspci | grep -E -i '(vga|3d)' | grep -i AMD > /dev/null; then
+	pacman --noconfirm -S vulkan-radeon lib32-vulkan-radeon xf86-video-amdgpu lib32-mesa libva-mesa-driver lib32-libva-mesa-driver mesa-vdpau lib32-mesa-vdpau
+fi
+
+# install Intel graphics
```

Comment:
i am not sure how safe this is to be installed alongside nvidia drivers. the arch wiki says it can cause problems.

Reply:
No as it states it checks which GPU is using and then install drivers for appropriate GPU